cmd/reconAccuracy/reconAccuracy.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/reconstruct"
+	"log"
+)
+
+//TODO: this code will need to change drastically for sequences of varying lengths.
+//The loop through the sequence is restricted by a single fasta and the tot calculation will need to calculate the total number of bps
+//ReconAccuracy calculates the total number of incorrectly reconstructed base pairs in a tree and returns a percentage of correct base calls
+func ReconAccuracy(simFilename string, reconFilename string, outFilename string) {
+	answer := reconstruct.ReconAccuracy(simFilename, reconFilename)
+	out := fileio.EasyCreate(outFilename)
+	defer out.Close()
+
+	for name, accuracy := range answer {
+		fmt.Fprintf(out, "%s %f \n", name, accuracy)
```

Comment:
Can you remove the space after the number and before the end of line?  Obviously not a huge deal, but some parsing functions will get confused and think there are three columns instead of two since they are two delimiters present (spaces) instead of only one.  The format string would be: "%s %f\n".  Another thing to consider is using a tab instead of a space.  In my experience this can make things a little easier since many of the Unix/Linux tools default to tab-separated columns.  \t is the magic <tab> character.  Kind of similar to \n.  The format string would be "%s\t%f\n"